OracleManagementAgent/README.md

```diff
@@ -0,0 +1,124 @@
+<!--
+    Copyright (c) 2022, Oracle and/or its affiliates.
+    Licensed under the Universal Permissive License v 1.0 as shown at
+    https://oss.oracle.com/licenses/upl.
+-->
+Oracle Management Agent on Docker
+=====
+This repository contains sample container configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux and Oracle OpenJDK 8.
+
+The certification of Oracle Management Agent on Docker does not require the use of any file presented in this
+repository. Customers and users are welcome to use them as starters, and customize/tweak, or create
+from scratch new scripts and Dockerfiles.
+
+
+## How to build and run
+
+Oracle Management Agent images use the Oracle Linux 7 Slim docker image as the base image.
+
+### Prerequisites
+
+* Latest Management Agent software for Linux (Linux-x86_64/latest/oracle.mgmt_agent.zip)
+
+#### Build steps
+
+1. [Download the Management Agent software](https://cloud.oracle.com/macs)
+
+    **Note: Select 'Downloads and Keys' then download 'Agent for LINUX (X86_64)' of the package type ZIP.**
+    ```shell
+    Copy-paste link: https://cloud.oracle.com/macs
+    ```
+
+1. Copy the downloaded bundle to the same directory as the Dockerfile
+
+    ```shell
+    $ cp oracle.mgmt_agent.zip OracleManagementAgent/dockerfiles/latest/
+    ```
+
+1. Change directory to build the dockerfile
+
+    ```shell
+    $ cd OracleManagementAgent/dockerfiles/latest/
+    ```
+
+1. Build the Docker image
+
+    ```shell
+    $ docker build -t oracle/mgmtagent-container .
+    ```
+
+1. [Create and Download the Install Key](https://docs.oracle.com/en-us/iaas/management-agents/doc/install-management-agent-chapter.html)
+
+    **Note: Follow the steps in the 'Create Install Key' and 'Configure a Response File' sections to create the install key and save it as input.rsp.**
+    ```shell
+    $ docker volume create mgmtagent-volume
+    ```
+
+1. Create a Docker volume to share configs with the container
+
+    ```shell
+    $ docker volume create mgmtagent-volume
+
+    # identify the mount point location to use in next steps
+    $ docker volume inspect mgmtagent-volume|grep Mountpoint
+        "Mountpoint": "/var/lib/docker/volumes/mgmtagent-volume/_data",
+    ```
+
+1. Copy the Install Key (input.rsp) into the shared Docker volume Mountpoint
+
+    ```shell
+    # create any necessary dirs
+    $ mkdir -p /var/lib/docker/volumes/mgmtagent-volume/_data/mgmtagent_secret
+    $ cp input.rsp /var/lib/docker/volumes/mgmtagent-volume/_data/mgmtagent_secret/
+    ```
+
+1. [Apply Prerequisites for Deploying Management Agents](https://docs.oracle.com/en-us/iaas/management-agents/doc/perform-prerequisites-deploying-management-agents.html)
+
+    **Note: Without the prerequisites applied the agent may not function as expected.**
+
+1. Start the Docker container
```

Comment:
Please rephrase this as "Use Docker to start a container"

Reply:
fixed